src/iot/feedTypes.js

```diff
@@ -0,0 +1,57 @@
+import { toCamelCase } from '../utils/objects';
+
+/**
+ * @typedef {Object} FeedType
+ * @property {String} createdAt
+ * @property {String} downAfter
```

Comment:
Can you include more information about this, like is it seconds? milliseconds? etc

Reply:
Had to do some digging for this one lol. It turned out to be seconds. I added a comment.